schedule/yam/agama_headless.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: agama_headless
+description: >
+  Perform Agama headless default installation.
+  A headless system is just boot to multi-user.target without a graphical environment.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/validate_headless
+  - yam/agama/patch_agama_tests
+  - yam/agama/agama
+  - installation/grub_test
+  - installation/first_boot
+  - yam/validate/validate_base_product
+  - yam/validate/validate_first_user
+  - yam/validate/validate_hostname
```

Comment:
this check is more specific, we don't change it during default installation, please drop.

Reply:
dropped.